sla/metrics.py

```diff
@@ -0,0 +1,110 @@
+import pingparsing
+import requests
+from skale.dataclasses.skaled_ports import SkaledPorts
+from skale.schain_config.ports_allocation import calc_schain_base_port
+
+from tools.configs import GOOD_IP
+from tools.helper import HEALTH_REQ_URL, PORT, logger
+
+
+def get_metrics_for_node(skale, node, is_test_mode):
+    host = GOOD_IP if is_test_mode else node['ip']
+    metrics = get_ping_node_results(host)
+    if not is_test_mode:
+        healthcheck = get_containers_healthcheck(host)
+        schains_check = check_schains_for_node(skale, node['id'])
+        metrics['is_offline'] = metrics['is_offline'] | healthcheck | schains_check
+
+    logger.info(f'Received metrics from node ID = {node["id"]}: {metrics}')
+    return metrics
+
+
+def check_schain(skale, schain, node_ip):
+    name = schain['name']
+    addr = "http://" + node_ip + ":" + str(schain['http_rpc_port'])
+    logger.info(f'\nChecking {name}, {addr}')
+
+    try:
+        block_number = skale.web3.eth.blockNumber
+        logger.debug(f"Current block number = {block_number}")
```

Comment:
this debug output seems useless - maybe it makes sense to add sChain endpoint or name to it

Reply:
I have sChain name output just before this one (with level info). I think there's no sense to duplicate it for debug output but anyway - done :)